### PR TITLE
Ingress controller docs

### PR DESCRIPTION
**Pull request by jkotalik**
WIP for ingress controller docs and samples.

### CODE REVIEW

File: docs/docfx/articles/kubernetes-ingress.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Kubernetes Ingress Controller
```
Comment (Tratcher) [inline docs/docfx/articles/kubernetes-ingress.md:1]: Also add this to the TOC
https://github.com/microsoft/reverse-proxy/blob/main/docs/docfx/articles/toc.yml

File: docs/docfx/articles/kubernetes-ingress.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Introduced: 1.0.0
```
Comment (Tratcher) [inline docs/docfx/articles/kubernetes-ingress.md:3]:
```suggestion
Introduced: 1.0.0 Preview 11
```

File: samples/KuberenetesIngress.Sample/Sample/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Sample backend
```
Comment (Tratcher) [inline samples/KuberenetesIngress.Sample/Sample/README.md:1]: Can the backend app be replaced by an existing docker image like this?
https://code.mendhak.com/docker-http-https-echo/

One less moving part to maintain.

**Comment by jkotalik** [comment]
Will fix https://github.com/microsoft/reverse-proxy/issues/870 and https://github.com/microsoft/reverse-proxy/issues/871

**Review by Tratcher — APPROVED** [review]

**Pull request merged**